feature-utils/poly-look/src/poly-look.js

```diff
@@ -22,7 +23,8 @@ export {
   SankeyDiagram,
   TreeMap,
   PolyChart,
-  ChipGroup,
+  Chip,
+  FilterChips,
```

Comment:
Is the naming OK? It sounds like you are going to filter chips... Maybe `FilteringChips`